tests/utils.go

```diff
@@ -2811,6 +2811,9 @@ func ExpectBatchWithValidatedSend(expecter expect.Expecter, batch []expect.Batch
 			}
 			expectFlag = true
 			sendFlag = false
+			if _, ok := batch[i].(*expect.BExp); !ok {
+				return nil, fmt.Errorf("ExpectBatchWithValidatedSend support only expect of type BExp")
```

Comment:
Can you add  a note about this on the docstring?
BTW, what is not supported, the one with the timeout?

Reply:
Yes, `BExpT` is not supported.
Do you mean adding it to the comment before the method? I don't think it is required. The method also doesn't support sequential sends and expects and it is not in the comment. I think the error messages are clear enough. If you will agree, I will leave it as is.

Reply:
I will not insist, but the function API description should provide details for the one who uses it.
Consider that it make take a poor tester 4h to detect this :)

Reply:
I believe out testers are not poor:)